torch/csrc/jit/passes/lower_tuples.cpp

```diff
@@ -170,9 +169,18 @@ void LowerAllTuples(std::shared_ptr<Graph>& graph) {
   EnsureNoTuples(graph->block());
 }
 
+void flattenPrintInputs(Node* n) {
```

Comment:
I am not sure what this is. Was this a bad merge? Doesn't seem related to this PR.